src/Microsoft.Framework.DesignTimeHost/ApplicationContext.cs

```diff
@@ -279,6 +282,24 @@ private bool ProcessMessage()
                         _waitingForDiagnostics.Add(message.Sender);
                     }
                     break;
+                case "Plugin":
```

Comment:
No need, it does the same thing and this follows the format the `ApplicationContext` already uses.